components/dashboard/TextArea/PostButtons.tsx

```diff
@@ -62,18 +63,25 @@ const PostButtons = ({
     );
   }, [notes, selectedNoteId]);
 
-  const saveNoteHandler = () => {
+  const getFromStore1 = async (selectedNoteId: string) => {
+    if (!selectedNoteId) return;
+    return get(selectedNoteId);
+  };
```

Comment:
This return type of this function is `void | string` which we shouldn't be doing. Instead of returning nothing, we can return `null`.